.goreleaser.yaml

```diff
@@ -143,10 +140,10 @@ app_bundles:
         dst: Contents/Info.plist
 
 nfpms:
-  - formats: [deb]
+  - formats: [deb, rpm]
     id: deb-arm64-wayland
     builds: [linux-arm64-wayland]
-    file_name_template: rio_{{ .Version }}-1_arm64_wayland.deb
+    file_name_template: rio_{{ .Version }}-1_arm64_wayland
```

Comment:
fwiw can also use `{{.ConventionalFileName}}_wayland` to use the conventional name of each format

Reply:
cc @vedantmgoyal9